third_party/lint/0001-enable-buildifier-test-without-sandbox.patch

```diff
@@ -0,0 +1,137 @@
+diff --git a/buildifier/buildifier.bzl b/buildifier/buildifier.bzl
+index 42445e7..69f3341 100644
+--- a/buildifier/buildifier.bzl
++++ b/buildifier/buildifier.bzl
+@@ -34,8 +34,24 @@ def buildifier(**kwargs):
+ def _buildifier_test_impl(ctx):
+     return [buildifier_impl_factory(ctx, test_rule = True)]
+ 
+-buildifier_test = rule(
++_buildifier_test = rule(
+     implementation = _buildifier_test_impl,
+     attrs = buildifier_attr_factory(True),
+     test = True,
+ )
++
++def buildifier_test(**kwargs):
++    """
++    Wrapper for the _buildifier_test rule. Adds tags to disable sandboxing.
++
++    Args:
++      **kwargs: all parameters for _buildifier_test
++    """
++    if kwargs.get("no-sandbox", False):
++        tags = kwargs.get("tags", [])
++        # Note: the "external" tag is a workaround for bazelbuild#15516.
++        for t in ["no-sandbox", "no-cache", "external"]:
++            if t not in tags:
++                tags.append(t)
++        kwargs["tags"] = tags
++    _buildifier_test(**kwargs)
+diff --git a/buildifier/internal/factory.bzl b/buildifier/internal/factory.bzl
+index 779c789..0ae3bc1 100644
+--- a/buildifier/internal/factory.bzl
++++ b/buildifier/internal/factory.bzl
+@@ -46,6 +46,10 @@ def buildifier_attr_factory(test_rule = False):
+         "verbose": attr.bool(
+             doc = "Print verbose information on standard error",
+         ),
```

Comment:
I don't see any starlark code for handling the `verbose` attribute.

Reply:
This was part of the existing attr factory code. I believe it's used here: https://github.com/bazelbuild/buildtools/blob/7186f635531bf1c395f246cc2687aecb56ff0572/buildifier/internal/factory.bzl#L125